list.go

```diff
@@ -121,15 +121,18 @@ func getContainers(context *cli.Context) ([]containerState, error) {
 		if item.IsDir() {
 			container, err := factory.Load(item.Name())
 			if err != nil {
-				return nil, err
+				fmt.Fprintf(os.Stderr, "load container %s: %v", item.Name(), err)
```

Comment:
These need to have trailing `\n` otherwise it messes up the header. Also, personally I would prefer if we printed them in the order we would've output them (but that's a stylistic thing and doesn't really matter).